model_analyzer/config/generate/quick_run_config_generator.py

```diff
@@ -537,6 +550,13 @@ def _create_default_model_run_config(
                                                   default_model_config,
                                                   default_perf_analyzer_config)
 
+        default_composing_model_configs = self._create_default_composing_model_configs(
+            model)
+
+        if default_composing_model_configs:
+            default_model_run_config.add_bls_composing_model_configs(
```

Comment:
this function is BLS-specific, but the code for the function above implies it can be ensemble or bls?

Reply:
The function above is a generic composing config creator, but this method cannot be reached for an ensemble config (it has a different default config generator method).